doc/articles/migrating-from-previous-releases.md

```diff
@@ -6,6 +6,42 @@ uid: Uno.Development.MigratingFromPreviousReleases
 
 This article details the migration steps required to migrate from one version to the next when breaking changes are being introduced.
 
+## Uno Platform 5.3
+
+Uno Platform 5.3 contains an improved template and Uno.SDK versioning, new default text font and Rider support.
```

Comment:
No mention of the HR indicator needed here? (just for reference to the HR Indicator doc)

Reply:
This feature has no migration impact.